### ISSUE TITLE
A few items for the /guide

### ISSUE THREAD

**Issue by nrooney** [w3c/Guide#26]
- [ ] manu training session, video

- [ ] ivan herman piece on how to use github

**Comment by LJWatson** [comment]
The video content will need a transcript (text version of the dialogue), real-time captions (synchronised text alternative to the dialogue), and (if the video contains important visual information) it will also need audio description or better still, a default soundtrack that describes what is shown on the screen clearly enough for someone to understand even if they can't see the screen.

Let me know if I can help with these things.

**Comment by chaals** [comment]
@LJWatson sure you could help, if we can find a reference to the video in question. I have a hunch that the video in question is from @msporny and will ask if he can shed any light...

**Comment by msporny** [comment]
> I have a hunch that the video in question is from @msporny and will ask if he can shed any light...

Yep, looking for a link now... might even have a transcription... brb.

**Comment by msporny** [comment]
Found it!

https://www.youtube.com/watch?v=0eQXU6Z-A6Q

**Comment by xfq** [comment]
The i18n WG also has:

- https://www.w3.org/International/i18n-activity/guidelines/issues.html
- https://www.w3.org/International/i18n-activity/guidelines/github
